### PR TITLE
Use native foreach around T-Regx

### PR DESCRIPTION
**Pull request by veelkoov**

### CODE REVIEW

File: tests/ByCodeAnalysis/ArtisanFieldsTest.php [hunk 1]
```diff
@@ -48,13 +49,16 @@ public function testArtisanTsModel(): void
 
         $fieldsInJson = Fields::public()->asArray();
 
-        $matches->forEach(function (Detail $detail) use (&$fieldsInJson): void {
+        foreach ($matches as $detail) {
             $field = array_shift($fieldsInJson);
 
-            static::assertNotNull($field);
+            /* Silence PHPStan */
+            static::assertInstanceOf(Field::class, $field);
```
Comment (veelkoov) [inline tests/ByCodeAnalysis/ArtisanFieldsTest.php:56]: Why is THIS needed NOW?

Comment (veelkoov) [reply tests/ByCodeAnalysis/ArtisanFieldsTest.php]: It wasn't.

File: tests/ByCodeAnalysis/ArtisanFieldsTest.php [hunk 2]
```diff
@@ -48,13 +48,15 @@ public function testArtisanTsModel(): void
 
         $fieldsInJson = Fields::public()->asArray();
 
-        $matches->forEach(function (Detail $detail) use (&$fieldsInJson): void {
+        foreach ($matches as $detail) {
+            static::assertInstanceOf(Detail::class, $detail); // Silence PHPStan
```
Comment (veelkoov) [inline tests/ByCodeAnalysis/ArtisanFieldsTest.php:52]: Outside tests I'd probably go with a `@var` annotation...

Comment (danon) [reply tests/ByCodeAnalysis/ArtisanFieldsTest.php:52]: Yes, my mistake. That's very good idea.
